storm-client/test/jvm/org/apache/storm/grouping/ShuffleGroupingTest.java

```diff
@@ -55,17 +55,17 @@ public void testShuffleGrouping() {
             List<Integer> taskIds = grouper.chooseTasks(inputTaskId, Lists.newArrayList());
 
             // Validate a single task id return
-            assertNotNull("Not null taskId list returned", taskIds);
-            assertEquals("Single task Id returned", 1, taskIds.size());
+            assertNotNull(taskIds, "Not null taskId list returned");
+            assertEquals(1, taskIds.size(), "Single task Id returned");
```

Comment:
error message should negate the assertion

Reply:
done